src/Symfony/Component/Mailer/Bridge/Amazon/composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=7.1.3",
-        "symfony/mailer": "^4.4|^5.0"
+        "symfony/mailer": "^4.4.18|^5.1.10"
```

Comment:
`"^4.4.18|~5.1.10|^5.2.1"` for all of them

Reply:
Ah damn you are right :-) 

Done